pkg/kubelet/prober/manager.go

```diff
@@ -207,6 +207,23 @@ func (m *manager) UpdatePodStatus(podUID types.UID, podStatus *api.PodStatus) {
 		}
 		podStatus.ContainerStatuses[i].Ready = ready
 	}
+	// init containers are ready if they have exited with success or if a readiness probe has
+	// succeeded.
+	for i, c := range podStatus.InitContainerStatuses {
```

Comment:
Since readiness check is not allowed for init containers, why do we want the prober to update the readiness for them?